recipes/juypterlab-language-packs/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "jupyterlab-language-packs" %}
+{% set version = "0.0.2.dev0" %}
+{% set build_number = 0 %}
+{% set locales = ["ar-SA", "no-NO"] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: ar-SA
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name[-1] }}-ar-SA/{{ name[-1] }}-ar-SA-{{ version }}.tar.gz
+    sha256: d0e4a6d9ceeee573761e34731a6426a193956f7102ddfd40ae04620edb3dd61d
+  - folder: no-NO
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name[-1] }}-no-NO/{{ name[-1] }}-no-NO-{{ version }}.tar.gz
+    sha256: 3f1bcaa8d8c40d5e1cb3051a78c584203ade87286cff492fc19878dc1027e9f5
+
+build:
+  number: {{ build_number }}
```

Don't template the build number. if not specified in the output, the top level one is taken.
```suggestion
  number: 0
```

---

Thanks for the tip